### DISCUSSION TITLE
Obtaining HOMO-LUMO Gap From .mol File

### DISCUSSION THREAD

**Discussion by Prometheus1400** [cclib/cclib#1060] [Q&A] [answered]
I will preface this by saying that I have no experience with computational chemistry. I am interested in obtaining the HOMO-LUMO gap for molecules in the PubChemQC dataset. I have been looking at the .mol files which have B3LYP 6-31G(d) gamess geometry optimization, and not much additional information. I can successfully read the file with ccread, but I have been unable to determine how to proceed from there. From the documentation it looked like CDA was what I may need, but I am not sure what the fragments are, or how to obtain them.   
Any help would be appreciated, thanks!

Note, here is the contents for a typical .mol file for reference if it is useful.
[075275021.txt](https://github.com/cclib/cclib/files/6904852/075275021.txt)

**Comment by ghutchis** [comment, marked as answer]
The PubChemQC mol files do not include any of the computed data besides the geometries. You need the corresponding `.log.xz` files. IIRC, there's a way to query their website for computed properties, but it's been a while.

**Reply by Prometheus1400** [reply]
Thanks for the reply!
You are right, there is a way to query their website but I don't think they have an official API. I also thank you for directing me to look at the log file as I was able to find the values I needed to compute the HOMO/LUMO gap in there.

**Reply by ghutchis** [reply]
Yes, `cclib` should be able to get everything you might want from the GAMESS log. It's too bad they don't add some properties to the `.mol` files, but 🤷‍♂️
